docs/site/Calling-other-APIs-and-Web-Services.md

```diff
@@ -9,12 +9,121 @@ summary:
 ---
 
 Your API implementation often needs to interact with REST APIs, SOAP Web
-Services or other forms of APIs.
+Services, gRPC microservices, or other forms of APIs.
 
-How to:
+To facilitate calling other APIs or web services, we introduce `@loopback/service-proxy`
+module to provide a common set of interfaces for interacting with backend services.
 
-- Set up a Service
-- Use services with controllers
-- Reuse models between services and controllers
+## Installation
 
-{% include content/tbd.html %}
+```
+$ npm install --save @loopback/service-proxy
+```
+
+## Usage
+
+### Define a data source for the service backend
+
+```ts
+import {DataSourceConstructor, juggler} from '@loopback/service-proxy';
+
+const ds: juggler.DataSource = new DataSourceConstructor({
```

Comment:
This may be outside of this pull request, but it would be great if consumers of `@loopback/service-proxy` did not need to know about `juggler` namespace.

```ts
import {DataSourceConstructor, DataSource} from '@loopback/service-proxy';
const ds: DataSource = new DataSourceConstructor({
  // ...
});
```

Ideally, we should not need two type entities (DataSource + DataSourceConstructor) and have a single entity behaving like a proper ES6 class:

```ts
import {DataSource} from '@loopback/service-proxy';
const ds = new DataSource({
  // ...
});
```